charts/kubearchive/templates/pre-install.yaml

```diff
@@ -0,0 +1,77 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: pre-install-view
+  annotations:
+    helm.sh/hook: pre-install
+    helm.sh/hook-delete-policy: hook-succeeded
+    helm.sh/hook-weight: "-10"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
Could we use two roles instead of a ClusterRole? Given that we know the namespaces, the less permissions we provide the better.

Reply:
This no longer matters, but this role only lives during the installation.